akka-docs-new/src/main/paradox/guide/modules.md

```diff
@@ -1,169 +1,171 @@
+While distributed as a JAR artifact, Remoting resembles a module more than it does a library. You enable it mostly
+with configuration, it has only a few APIs. Thanks to the actor model, a remote and local message send looks exactly the
+same. The patterns that you use on local systems translate directly to remote systems.
 You will rarely need to use Remoting directly, but it provides the foundation on which the Cluster subsystem is built.
- 
-Some of the problems Remoting solves are
 
-* How to address actor systems living on remote hosts?
-* How to address individual actors on remote actor systems?
-* How to turn messages to bytes on the wire?
-* How to manage low-level, network connections (and reconnections) between hosts, detect crashed actor systems and hosts, 
-  all transparently?
-* How to multiplex communications from unrelated set of actors on the same network connection, all transparently?
+Some of the challenges Remoting solves are:
+
+* How to address actor systems living on remote hosts.
+* How to address individual actors on remote actor systems.
+* How to turn messages to bytes on the wire.
+* How to manage low-level, network connections (and reconnections) between hosts, detect crashed actor systems and hosts,
+  all transparently.
+* How to multiplex communications from an unrelated set of actors on the same network connection, all transparently.
 
 ### Cluster
 
-If you have a set of actor systems that cooperate to solve some business problem, then you likely want to manage these set of 
-systems in a disciplined way. While Remoting solves the problem of addressing and communicating with components of 
+If you have a set of actor systems that cooperate to solve some business problem, then you likely want to manage these set of
+systems in a disciplined way. While Remoting solves the problem of addressing and communicating with components of
 remote systems, Clustering gives you the ability to organize these into a "meta-system" tied together by a membership
-protocol. **In most cases, you want to use the Cluster module instead of using Remoting directly.** 
-Cluster provides an additional set of services on top of Remoting that most real world applications need. 
+protocol. **In most cases, you want to use the Cluster module instead of using Remoting directly.**
+Clustering provides an additional set of services on top of Remoting that most real world applications need.
 
-The problems the Cluster module solves (among others) are
+The challenges the Cluster module solves, among others, are:
 
-* How to maintain a set of actor systems (a cluster) that can communicate with each other and consider each other as part of the cluster?
-* How to introduce a new system safely to the set of already existing members?
-* How to detect reliably systems that are temporarily unreachable?
+* How to maintain a set of actor systems (a cluster) that can communicate with each other and consider each other as part of the cluster.
```

Comment:
Do you mean why I decided to remove `?`? I see them as statements/facts more than questions.